doc/docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,37 @@
+
+= Neo4j Spark Connector v{docs-version}
+:toc:
+:toclevels: 5
+:sectids:
+:sectlinks:
+:sectnums:
+:sectnumlevels: 5
+:env-docs: true
+
+ifdef::backend-html5[(C) {copyright}]
+
+License: link:{common-license-page-uri}[Creative Commons 4.0]
+
+[abstract]
+--
+This is the user guide for Neo4j Spark Connector version {docs-version}, authored by the Neo4j Labs Team.
+--
+
+The guide covers the following areas:
+
+* <<_project_overview>> -- Project overview
+* <<_quick_start>> -- Get Started Fast with the most Common Scenarios
+* <<neo4j_causal_cluster>> -- Using with Neo4j Causal Cluster
+* <<cloud>> -- Configuring a connection to a Confluent Cloud
+* <<faq>> -- Neo4j Streams FAQ
+
+
+include::overview.adoc[]
```

Comment:
I don't think we need to include each pages in the index. It will be possible to navigate between each pages using the left navigation.